app/src/androidTest/java/io/github/polysmee/appointments/AppointmentActivityTest.java

```diff
@@ -33,29 +52,77 @@
 import static com.schibsted.spain.barista.interaction.BaristaEditTextInteractions.writeTo;
 import static com.schibsted.spain.barista.interaction.BaristaPickerInteractions.setDateOnPicker;
 import static com.schibsted.spain.barista.interaction.BaristaPickerInteractions.setTimeOnPicker;
+import static com.schibsted.spain.barista.interaction.BaristaSleepInteractions.sleep;
 import static org.hamcrest.Matchers.allOf;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 
 @RunWith(AndroidJUnit4.class)
 public class AppointmentActivityTest {
     String title = "title";
     String course = "course";
-    String startTime = "23/03/2021   -   17:02";
-    String endTime = "23/03/2021   -   18:02";
+    String startTime = "23/03/2022   -   17:02";
+    String endTime = "23/03/2022   -   18:02";
+
+    private static final String username1 = "Mathis L'utilisateur";
+    private static final String id2 = "-SFDkjsfdl";
+    private static final String id3 = "-SFDkjsfdkwefwef";
+    private static final String username2 = "Sami L'imposteur";
+    private static final String username3 = "Leo le testeur";
+    private static final String id4 = "-SFDkjsfdl";
+    private static final String id5 = "-SFDkjsfdkwefwef";
+    private static final String username4 = "Thomas";
+    private static final String username5 = "Adrien";
+
+    private static final String appointmentTitle = "It's a title";
+    private static final String appointmentId = "-lsdqfkhfdlksjhmf";
+    private static final String appointmentCourse = "Totally not SWENG";
+    private static final long appointmentStart = 265655445;
+
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        DatabaseFactory.setTest();
+        AuthenticationFactory.setTest();
+        FirebaseApp.clearInstancesForTest();
+        FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext());
+        Tasks.await(AuthenticationFactory.getAdaptedInstance().createUserWithEmailAndPassword("polysmee134@gmail.com", "fakePassword"));
```

Comment:
When doing your tests, you choose an email address others haven't chosen themselves.